internal/namespaces/registry/v1/custom_tag.go

```diff
@@ -44,14 +44,14 @@ func tagGetBuilder(c *core.Command) *core.Command {
 			ImageID: tag.ImageID,
 		})
 		if err != nil {
-			return getTagResp, nil
+			return getTagResp, err
```

Comment:
```suggestion
			return getTagResp, nil
```

Reply:
This seems done on purpose to not make the command fail if you can't get additional info.